cgroups/cgroups.go

```diff
@@ -6,6 +6,33 @@ import (
 	"github.com/docker/libcontainer/devices"
 )
 
+type Manager interface {
+	// Apply cgroup configuration to the process with the specified pid
+	Apply(pid int) error
+
+	// Returns the PIDs inside the cgroup set
+	GetPids() ([]int, error)
+
+	// Returns statistics for the cgroup set
+	GetStats() (*Stats, error)
+
+	// Toggles the freezer cgroup according with specified state
+	Freeze(state FreezerState) error
+
+	// Destroys the cgroup set
+	Destroy() error
+
+	// Paths maps cgroup subsystem to path at which it is mounted.
```

Comment:
NewCgroupManager() and LoadCgroupManager() require Paths and Cgroups attributes. I have updated the comment.